sdk/lib/rtl/image.c

```diff
@@ -460,25 +460,27 @@ LdrProcessRelocationBlockLongLong(
 ULONG
 NTAPI
 LdrRelocateImage(
-    IN PVOID BaseAddress,
-    IN PCCH  LoaderName,
-    IN ULONG Success,
-    IN ULONG Conflict,
-    IN ULONG Invalid)
+    _In_ PVOID BaseAddress,
+    _In_opt_ PCCH LoaderName,
+    _In_ ULONG Success,
+    _In_ ULONG Conflict,
+    _In_ ULONG Invalid)
 {
     return LdrRelocateImageWithBias(BaseAddress, 0, LoaderName, Success, Conflict, Invalid);
 }
 
 ULONG
 NTAPI
 LdrRelocateImageWithBias(
-    IN PVOID BaseAddress,
-    IN LONGLONG AdditionalBias,
-    IN PCCH  LoaderName,
-    IN ULONG Success,
-    IN ULONG Conflict,
-    IN ULONG Invalid)
+    _In_ PVOID BaseAddress,
+    _In_ LONGLONG AdditionalBias,
+    _In_opt_ PCCH LoaderName,
+    _In_ ULONG Success,
+    _In_ ULONG Conflict,
+    _In_ ULONG Invalid)
 {
+    UNREFERENCED_PARAMETER(LoaderName);
```

Comment:
Move this after the variables declarations, please.

Reply:
Committed.